salesforce_api/gsheet-new-row-to-sfdc-new-contact/sample.bal

```diff
@@ -0,0 +1,78 @@
+import ballerina/log;
+import ballerinax/googleapis.sheets;
+import ballerinax/salesforce as sfdc;
+
+// Types
+type GSheetOAuth2Config record {
+    string clientId;
+    string clientSecret;
+    string refreshToken;
+    string refreshUrl = "https://www.googleapis.com/oauth2/v3/token";
+};
+
+type SalesforceOAuth2Config record {
+    string clientId;
+    string clientSecret;
+    string refreshToken;
+    string refreshUrl = "https://login.salesforce.com/services/oauth2/token";
+};
+
+// Constants
+const int HEADINGS_ROW = 1;
+
+// Google sheets configuration parameters
+configurable string spreadsheetId = ?;
+configurable string worksheetName = ?;
+configurable string duplicateWorksheetName = ?;
+configurable GSheetOAuth2Config GSheetOAuthConfig = ?;
+
+// Salesforce configuration parameters
+configurable SalesforceOAuth2Config salesforceOAuthConfig = ?;
+configurable string salesforceBaseUrl = ?;
+
+
+
```

Comment:
Remove extra new lins.

Reply:
Updated in [900e154](https://github.com/ballerina-guides/integration-samples/pull/31/commits/900e154cf4d53db6e093f4fd6d5b450900bbafa3)